src/test/java/teammates/test/cases/webapi/CreateCourseActionTest.java

```diff
@@ -33,35 +34,30 @@ public void testExecute() throws Exception {
         String instructorId = instructor1OfCourse1.googleId;
         String courseId = instructor1OfCourse1.courseId;
 
-        String[] submissionParams = new String[] {
-                Const.ParamsNames.INSTRUCTOR_ID, instructorId,
-                Const.ParamsNames.COURSE_ID, "new-course",
-                Const.ParamsNames.COURSE_NAME, "New Course",
-                Const.ParamsNames.COURSE_TIME_ZONE, "UTC",
-        };
+        CourseCreateRequest courseCreateRequest = new CourseCreateRequest();
+        courseCreateRequest.setCourseName("New Course");
+        courseCreateRequest.setTimeZone("UTC");
+        courseCreateRequest.setCourseId("new-course");
 
         ______TS("Typical case with new course id");
 
-        if (CoursesLogic.inst().isCoursePresent("new-course")) {
+        if (logic.isCoursePresent("new-course")) {
             CoursesLogic.inst().deleteCourseCascade("new-course");
         }
 
         loginAsInstructor(instructorId);
-        AddInstructorCourseAction action = getAction(submissionParams);
+        CreateCourseAction action = getAction(courseCreateRequest);
         JsonResult result = getJsonResult(action);
 
         assertEquals(HttpStatus.SC_OK, result.getStatusCode());
 
         ______TS("Typical case with existing course id");
 
-        submissionParams = new String[] {
-                Const.ParamsNames.INSTRUCTOR_ID, instructorId,
-                Const.ParamsNames.COURSE_ID, courseId,
-                Const.ParamsNames.COURSE_NAME, "Existing Course",
-                Const.ParamsNames.COURSE_TIME_ZONE, "UTC",
-        };
+        courseCreateRequest.setCourseName("Existing Course");
+        courseCreateRequest.setTimeZone("UTC");
+        courseCreateRequest.setCourseId(courseId);
 
-        action = getAction(submissionParams);
+        action = getAction(courseCreateRequest);
         result = getJsonResult(action);
         MessageOutput message = (MessageOutput) result.getOutput();
```

Comment:
You mean above, right? In this case the course isn't created but instead a course with the same ID as an existing course is attempted to be created. Unless you didn't make a mistake and git force pushing messed it up.

Reply:
> instead a course with the same ID as an existing course is attempted to be created

If you choose the courseId carefully, this will not happen.

> Unless you didn't make a mistake and git force pushing messed it up.

How does this relate to `git push --force`?